src/queries.js

```diff
@@ -1,22 +1,32 @@
-import { gql, useMutation } from '@apollo/client';
+import { gql } from '@apollo/client';
 
 // get all selected=true users (id, name, profile, genre)
 
-// get all users (id, name, profile, song_title)
+// get all users
+const GET_ALL_USERS = gql`
+  {
+    fetchUsers {
+      id
+      name
+      profile
+      songTitle
+    }
+  }
+`;
 
 // get user by id (id, name, email, genre, song_title, profile, video, selected)
 
 // create mutation: add new user (name, email, genre, song_title, profile, video)
-const ADD_NEW_USER = gql`
-  mutation AddNewUser(
+const CREATE_USER = gql`
+  mutation CreateUser(
     $name: String!
     $email: String!
     $genre: String!
     $songTitle: String!
     $profile: Sting!
     $video: String!
   ) {
-    addNewUser(
+    createUser(
       name: $name,
       email: $email,
       genre: $genre,
```

Comment:
Looks like the backend will let us know if/when we need to change the syntax for our mutation.